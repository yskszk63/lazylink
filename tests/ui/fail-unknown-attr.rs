#[lazylink::lazylink(aaa="bbb")]
mod libc {
    extern "C" {
        fn puts(v: *const i8);
    }
}

fn main() {}
