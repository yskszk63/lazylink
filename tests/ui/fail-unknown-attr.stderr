error: unknown name
 --> $DIR/fail-unknown-attr.rs:1:25
  |
1 | #[lazylink::lazylink(aaa="bbb")]
  |                         ^
